utils/sql-db-seeder/src/utilisation-report/utilisation-report.helper.ts

```diff
@@ -99,3 +101,28 @@ export const createUploadedReport = (
 
   return utilisationReport;
 };
+
+export const createFeeRecordsForReport = (): FeeRecordEntity[] => {
+  const feeRecordWithMatchingCurrencies = createFeeRecord({
+    facilityId: '12345678',
```

Comment:
This might be a bit complicated, but have we thought about matching these to actual facilities so that when we get to adding the keying sheet we can fetch data about the facility?

Reply:
If I match it to an actual facility, I would probably then need to worry about properly mapping the data as well which I'd rather not do at this stage. For now, I just want some simple data in there so that I can actually view the page and the table, and we can update the seeder as we advance the keying sheet functionality

Reply:
Ok we can leave to when we add keying sheets, just worth being aware we are likely to make changes to this when we are writing tests since they run the seeding job